src/test/TestDASHTree.cpp

```diff
@@ -49,10 +49,11 @@ TEST_F(DASHTreeTest, CalculateBaseDomain)
 
 TEST_F(DASHTreeTest, CalculateEffectiveUrlFromRedirect)
   {
-    // like base_url_, effective_url_ should be path, not including filename
+    // like base_url_, base_url_ should be path, not including filename
```

Comment:
Can probably remove this comment now and rename test to CalculateBaseUrlFromRedirect?

Reply:
done :)